tests/usb_packet.py

```diff
@@ -68,46 +69,56 @@
 RXA_START_DELAY = 5 #Taken from RTL sim
 RX_RX_DELAY = 40
 
-PID_DATA1 = 0xb
-PID_DATA0 = 0x3
-
-
-
-def AppendSetupToken(packets, ep, address, **kwargs):
-    ipg = kwargs.pop('inter_pkt_gap', 500)
-    AppendTokenPacket(packets, 0x2d, ep, ipg, address, **kwargs)
-
-def AppendOutToken(packets, ep, address, **kwargs):
-    ipg = kwargs.pop('inter_pkt_gap', 500) 
-    AppendTokenPacket(packets, 0xe1, ep, ipg, address, **kwargs)
-
-def AppendPingToken(packets, ep, address, **kwargs):
-    ipg = kwargs.pop('inter_pkt_gap', 500) 
-    AppendTokenPacket(packets, 0xb4, ep, ipg, address, **kwargs)
-
-def AppendInToken(packets, ep, address, **kwargs):
+#TODO shoud we have a PID class?
+#TODO remove the inverted check bits 
+USB_PID = {
+            "OUT"       : 0xE1,
+            "IN"        : 0x69,
+            "SETUP"     : 0x2D,
+            "SOF"       : 0xA5,
+            "DATA1"     : 0x4b,
+            "DATA0"     : 0xc3,
+            "ACK"       : 0xD2,
+            "PING"      : 0xB4,
+            "RESERVED"  : 0x0F,
+            "NAK"       : 0x5A,
+        }
+
+#def AppendSetupToken(packets, ep, address, **kwargs):
+#    ipg = kwargs.pop('inter_pkt_gap', 500)
+#    AppendTokenPacket(packets, 0x2d, ep, ipg, address, **kwargs)
+
+#def AppendOutToken(packets, ep, address, **kwargs):
+#    ipg = kwargs.pop('inter_pkt_gap', 500) 
+#    AppendTokenPacket(packets, 0xe1, ep, ipg, address, **kwargs)
+
+#def AppendPingToken(packets, ep, address, **kwargs):
+#    ipg = kwargs.pop('inter_pkt_gap', 500) 
+#    AppendTokenPacket(packets, 0xb4, ep, ipg, address, **kwargs)
+
+#def AppendInToken(packets, ep, address, **kwargs):
     #357 was min IPG supported on bulk loopback to not nak
     #lower values mean the loopback NAKs
-    ipg = kwargs.pop('inter_pkt_gap', 10) 
-    AppendTokenPacket(packets, 0x69, ep, ipg, address, **kwargs)
+#    ipg = kwargs.pop('inter_pkt_gap', 10) 
+#    AppendTokenPacket(packets, 0x69, ep, ipg, address, **kwargs)
 
-def AppendSofToken(packets, framenumber, **kwargs):
-    ipg = kwargs.pop('inter_pkt_gap', 500) 
+#def AppendSofToken(packets, framenumber, **kwargs):
+#    ipg = kwargs.pop('inter_pkt_gap', 500) 
     
     # Override EP and Address 
-    ep = (framenumber >> 7) & 0xf
-    address = (framenumber) & 0x7f
-    AppendTokenPacket(packets, 0xa5, ep, ipg, address, **kwargs)
-
-def AppendTokenPacket(packets, _pid, ep, ipg, addr=0, **kwargs):
-    
-    data_valid_count = kwargs.pop('data_valid_count', 0)
-    packets.append(TokenPacket( 
-        inter_pkt_gap=ipg, 
-        pid=_pid,
-        address=addr, 
-        endpoint=ep,
-        data_valid_count=data_valid_count))
+#    ep = (framenumber >> 7) & 0xf
+#    address = (framenumber) & 0x7f
+#    AppendTokenPacket(packets, 0xa5, ep, ipg, address, **kwargs)
```

Comment:
Any reason not to delete all this commented-out code?

Reply:
Done